tests/v2prov/cluster/clusters.go

```diff
@@ -90,6 +90,19 @@ func New(clients *clients.Clients, cluster *provisioningv1api.Cluster) (*provisi
 				DisableSnapshots: true,
 			}
 		}
+
+		// If an override is specified for a data directory, and one is not explicitly set for the cluster, use it
+		if dir := defaults.DistroDataDir; cluster.Spec.RKEConfig.DataDirectories.K8sDistro == "" {
+			cluster.Spec.RKEConfig.DataDirectories.K8sDistro = dir
+		}
```

Comment:
nit: to make it easier to understand, same to the following two ifs. 
```suggestion
		if cluster.Spec.RKEConfig.DataDirectories.K8sDistro == "" {
			cluster.Spec.RKEConfig.DataDirectories.K8sDistro = defaults.DistroDataDir
		}
```